src/pages/MyPatches.tsx

```diff
@@ -96,32 +109,48 @@ export const MyPatches = () => {
           data-cy="commit-queue-checkbox"
           onChange={onCheckboxChange}
           label="Show Commit Queue"
-          checked={getQueryVariables(search).$includeCommitQueue}
+          checked={getQueryVariables(search).includeCommitQueue}
         />
       </FiltersWrapperSpaceBetween>
+      <Pagination
+        onChange={onChange}
+        current={page}
+        pageSize={LIMIT}
+        total={get(data, "userPatches.filteredPatchCount", 0)}
+      />
+      <>
+        {error ? (
+          <ErrorWrapper>{error}</ErrorWrapper>
+        ) : loading ? (
+          <StyledSkeleton active={true} title={false} paragraph={{ rows: 4 }} />
+        ) : get(data, "userPatches.patches", []).length !== 0 ? (
+          data.userPatches.patches.map((p) => <PatchCard key={p.id} {...p} />)
+        ) : (
+          <NoResults data-cy="no-patches-found">No patches found</NoResults>
+        )}
```

Comment:
can we clean this up a little bit and maybe move some logic out. Its difficult to read when we have 3 layers deep of ternaries